bridge/hardhat.config.js

```diff
@@ -73,7 +74,7 @@ module.exports = {
       gasPrice: 20000000000,
       hardfork: 'istanbul', // London hardfork is incompatible with RSK gasPrice
       tags: ['integrationTest', 'local'],
-      saveDeployments: false,
+      saveDeployments: true,
```

Comment:
keep this in false as we want it to be deployed always for integrationTest

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/267/commits/e557c97710c5051828bf529a1c4b1bb23f97709d